From a6f6e1a994f5ec6533c350a9de57bc8cfe9e57f6 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Thu, 31 Aug 2017 21:29:17 -0700
Subject: [PATCH 2626/2748] HADOOP-14824. Update ADLS SDK to 2.2.2 for MSI
 fix. Contributed by Atul Sikaria.

(cherry picked from commit 7ecc6dbed62c80397f71949bee41dcd03065755c)

Change-Id: I7eef115b9be7e711b5949bfdbafa76cf5aa65e17
---
 hadoop-tools/hadoop-azure-datalake/pom.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-tools/hadoop-azure-datalake/pom.xml b/hadoop-tools/hadoop-azure-datalake/pom.xml
index 5f782d5..315742c 100644
--- a/hadoop-tools/hadoop-azure-datalake/pom.xml
+++ b/hadoop-tools/hadoop-azure-datalake/pom.xml
@@ -111,7 +111,7 @@
     <dependency>
       <groupId>com.microsoft.azure</groupId>
       <artifactId>azure-data-lake-store-sdk</artifactId>
-      <version>2.2.1</version>
+      <version>2.2.2</version>
     </dependency>
     <!--  ENDS HERE-->
     <dependency>
-- 
1.7.9.5

